### PR TITLE
New Color Prop for Carousels with Peek

### PR DESCRIPTION
**Pull request by bcfancher-mc**
## Overview
adding the ability to set the gradient color for peeking

## Risks
Low - default color is still black.  Should see no observable difference in behavior without this new prop.

## Changes
image

## Issue
N/A

## Breaking change?
Backwards Compatible

### CODE REVIEW

File: src/components/Carousel/index.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ export default class Carousel extends PureComponent {
     sliderRef: PropTypes.func,
     transition: PropTypes.string,
     variableWidth: PropTypes.bool,
+    color: PropTypes.string,
```
Comment (stephenp) [inline src/components/Carousel/index.js:79]: I think this is supposed to be `colorForThatFantasticFunkInPeekMode`

File: src/components/Carousel/index.js [hunk 2]
```diff
@@ -167,6 +170,26 @@ export default class Carousel extends PureComponent {
       'mc-carousel__slider': true,
     })
 
+    let peekStyles
+    if (peek) {
+      peekStyles = {
```
Comment (stephenp) [inline src/components/Carousel/index.js:175]: I think you'll only need to set the `background` styles here if everything else is already coming from the modifiers.scss file. We do this on the `TileOverlay` component: https://github.com/yankaindustries/mc-components/blob/master/src/components/TileOverlay/index.js

Comment (stephenp) [reply src/components/Carousel/index.js:175]: Disregard - I remember the convo in slack now about not being able to style `:after` pseudo elements.

File: src/components/Carousel/index.js (hunk 1 above)
Comment (bcfancher-mc) [reply src/components/Carousel/index.js:79]: dammit!  good catch

**Pull request merged**